tests/integration/src/test/resources/org/mule/test/integration/locator/component-locator-config.xml

```diff
@@ -14,5 +14,10 @@
         </async>
     </flow>
 
+    <flow name="anotherFlow">
+        <scheduler/>
```

Comment:
it works as it used to work with <poll> I don't why it would be a problem to have a default value for the frequency.